cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -31,6 +31,15 @@
     .no-js &:not( .o-featured-content-module_visual ) {
         border: 1px solid @gray-40;
     }
+
+    &_iframe-container {
```

Comment:
You need to remove the redundant code, so the `o-video-player_iframe-container` rule further down.

Reply:
https://github.com/cfpb/consumerfinance.gov/blob/8899c4e586a79fa22649ad4c5012606fa1ebc4b0/cfgov/unprocessed/css/organisms/video-player.less#L230-L233

Reply:
As is, this'll produce 

```
.o-video-player .o-video-player_iframe-container { 
    height: 100%;
    margin: 0;
} 

.o-video-player_iframe-container { 
    height: 100%;
    margin: 0;
}
```

Reply:
Resolved in https://github.com/cfpb/consumerfinance.gov/pull/6708/commits/e649ddb0fcaca91324ad6f45ca71c4551f4cf717